### PR TITLE
#348 Add profile links to lms features

### PR DESCRIPTION
**Pull request by MahoMori**
## Proposed changes

This resolves issue #348.

Add links to Journal and Competency features to LMS on the profile page. The Competency button linked to Competency page and the journal button linked to home page.
We did not create a link to progress bar as it will be on the profile page after #349 is solved.

## Screenshot
Screenshot 2022-05-19 135448

## Checklist

- [x] Are the issues being addressed linked to this PR?
- [x] Do all commit messages start with the issue number?
- [x] Are all code changes sufficiently tested?
- [x] Are there screenshots for UI changes?

### CODE REVIEW

File: webapp/src/components/Profile.tsx [hunk 1]
```diff
@@ -27,95 +31,119 @@ const user = {
 };
 
 const Profile = () => {
+  let navigate = useNavigate();
```
Comment (achichikalova) [inline webapp/src/components/Profile.tsx:34]: Why do we use the UseNavigation hook here?
What if use button component with href attribute instead:
`<Button
            component="a"
            sx={{ mr: 1, height: '40px' }}
            href={'/competencies'}
  >Competencies</Button>`

**Review by denizedaza — APPROVED** [review]
Ran the branch on my local machine, it works as intended and matches the ticket issue. The team already made a note that the progress button is missing and will be implemented in a future ticket. Also noticed linting fixes within code. 
Noted that package.json & yarn.lock files edited to account for Windows users and doesn't pose an issue when running on my machine (Mac)

**Review by achichikalova — CHANGES_REQUESTED** [review]
Could you consider my suggestion with the Button component instead of using useNavigation hook, please? Thank you. :)
Good job!

File: webapp/src/components/Profile.tsx (hunk 1 above)
Comment (MahoMori) [reply webapp/src/components/Profile.tsx:34]: I checked if the way you suggested works and it does. We will discuss which way is better with others on Friday.

Comment (MahoMori) [reply webapp/src/components/Profile.tsx]: I changed useNavigate hook to buttons with href attribute

**Review by achichikalova — CHANGES_REQUESTED** [review]
Why are we wrapping the container with the <div> element?
We already have the Container element, we don't need the <div> element here.
Also, to be more consistent with the code, I suggest deleting the <div> wrapping.
Thank you.

**Review by achichikalova — APPROVED** [review]
All issues were resolved. 
The code is consistent with the whole project and works well.

**Pull request merged**